content/spin/redis-trigger.md

```diff
@@ -23,54 +30,154 @@ The Redis instance address is specified in the application trigger:
 trigger = { type = "redis", address = "redis://localhost:6379" }
 ```
 
-> We are [exploring adding authentication for connecting to Redis](https://github.com/fermyon/spin/issues/192)
-> and [adding host support for connecting to Redis databases](https://github.com/fermyon/spin/issues/181),
-> from components, which would allow using the key/value store and publishing
-> messages to channels.
+The Redis trigger also requires an `address` field.  This is the address of the Redis instance to monitor.  Specify it using the `redis:` URL scheme.
+
+> If you create an application from a Redis template, the trigger will be already set up for you.
+
+By default, Spin does not authenticate to Redis. You can work around this by providing a password in the `redis://` URL.  For example: `address = "redis://:p4ssw0rd@localhost:6379"`
```

Comment:
Do we need some *ALERT* here to remind users to not commit password to a VCS if they embed them in the address URL like such?